guides/moving-to-v2.md

```diff
@@ -8,11 +8,13 @@ is Coder's open core remote development platform first launched in June 2022.
 This document shares best practices for moving your workflows from Coder v1 to
 Coder v2.
 
+![Coder v2 Dashboard](https://raw.githubusercontent.com/coder/coder/main/docs/images/hero-image.png)
+
 > If you are current Coder v1 customer and to try Coder v2, [we'd like to hear from
 > you](https://calendly.com/bpmct/30min) to help inform our roadmap and
```

Comment:
I do not like the 1:1, single-point-of-failure aspect of a calend.ly. Replace with support@coder.com so it lands in Slack and we have a record and can reply all to store the thread in Zendesk.

Reply:
Agreed.